pkgs/shells/fish/default.nix

```diff
@@ -237,14 +237,13 @@ let
              "$out/share/fish/functions/__fish_print_help.fish"
       sed -e "s|clear;|${getBin ncurses}/bin/clear;|"      \
           -i "$out/share/fish/functions/fish_default_key_bindings.fish"
-      sed -e "s|python3|${getBin python3}/bin/python3|"    \
```

Comment:
Could you please add to the commit message **why** this change is done?

Just looking at the diff looks like it may break stuff. I unearthed (interestingly, from [your own IRC conversation](https://logs.nix.samueldr.com/nixos/2020-04-28#3380835;)) that simply not supplying any `python` will [skip generating completions](https://github.com/fish-shell/fish-shell/blob/2bc4228d42919fc6600f3e4c21d3a0c7dff05843/share/functions/__fish_config_interactive.fish#L107). May be good to explain that as well, for whoever will stumble over this for whatever reason in the future. I'm sure endlessly thankful if I encounter the rationale for weird-looking changes (especially deletions) in context.

Reply:
You're absolutely right; if I feel the need to explain it in the PR body it should also be in the commit message.

Note that what you're referring to there is not current anymore since I patched fish before opening this PR.